microsite/data/plugins/todo.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: TODO
+author: Spotify
+authorUrl: https://github.com/spotify
+category: Discovery
+description: Browse TODO comments in your project's source code.
+documentation: https://github.com/backstage/backstage/tree/master/plugins/todo
+iconUrl: https://backstage.io/img/todo-logo.png
```

Comment:
that's a 404 and isn't added in this pr right?

Reply:
Yep, it's there now :grin: